builder-support/dockerfiles/Dockerfile.target.centos-8

```diff
@@ -0,0 +1,17 @@
+# First do the source builds
+@INCLUDE Dockerfile.target.sdist
+
+# This defines the dstribution base layer
+# Put only the bare minimum of common commands here, without dev tools
+FROM centos:8 as dist-base
+RUN yum install -y epel-release
+RUN dnf -y install dnf-plugins-core
+RUN dnf config-manager --set-enabled powertools
+
+# Do the actual rpm build
+@INCLUDE Dockerfile.rpmbuild
+
+# Do a test install and verify
+# Can be skipped with skiptests=1 in the environment
+#@EXEC [ "$skiptests" = "" ] && include Dockerfile.rpmtest
```

Comment:
Is this commented on purpose?

Reply:
I just based it on the Centos-7 target Dockerfile, so the comments are from there. So no I didn't add the comments.